### ISSUE TITLE
feature request: final touches for instance switching

### ISSUE THREAD

**Issue by Rexogamer** [revoltchat/rvmob#20]
### What do you want to see?

- [ ] Automatically reload the app when the user changes the instance URL - might need to use something like `react-native-restart`
- [ ] Detect invite links for other known instances - should this also check whether the instance is the current one?
- [ ] Potentially make it clearer what instance you're logging into on the log in screen?

**Comment by MasicoreLord** [comment]
Wouldn't it be better for instances you don't know of a different invite path, to assume it's at `appUrl/invite/code`?

**Comment by Rexogamer** [comment]
It dies this already to an extent, but instances may also define their own paths

**Comment by MasicoreLord** [comment]
True, and wish if possible such paths were configurable in a way that other clients can recognize them, similar to how it could automatically configure stuff like the websocket url.
